src/NewCommand.php

```diff
@@ -160,4 +162,35 @@ protected function findComposer()
 
         return 'composer';
     }
+
+    /**
+     * Make it possible to interrupt execution in Windows mintty.
+     *
+     *@return void
+     */
+    protected function setSignalHandler()
+    {
+        if (function_exists('\sapi_windows_set_ctrl_handler')) {
+            // Windows - requires php 7.4
+            \sapi_windows_set_ctrl_handler([$this, 'SignalHandlerWindows']);
+        }
+    }
+
+    /**
+     * Handle CTRL+C / CTRL+break signal on Windows.
+     *
+     * @param  int  $signal
+     * @return void
+     */
+    public function SignalHandlerWindows(int $signal)
+    {
+        switch ($signal) {
+            case PHP_WINDOWS_EVENT_CTRL_C:
+            case PHP_WINDOWS_EVENT_CTRL_BREAK:
+                exit;
```

Comment:
do we not want to exit with a failure code?

Reply:
hi @GrahamCampbell, let me check on Linux what the return code is.

Reply:
on Linux, exit code is 130 which corresponds to Script terminated by Control-C. I'll make the change to have the Windows equivalent.

Reply:
There doesn't seem to be a set value for the exit code on Windows when the program is interrupted. At least I couldn't find it, therefore I put the 130 value as on Linux. Feel free to comment on this if you have a better idea.